packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/VisXZoom/VisXZoom.js

```diff
@@ -149,28 +162,31 @@ function VisXZoom({
       {_zoom => {
         zoomRef.current = _zoom
         return (
-          <ZoomingComponent
-            initialTransformMatrix={_zoom.initialTransformMatrix}
-            transformMatrix={_zoom.transformMatrix}
-            transform={_zoom.toString()}
-            {...props}
-          >
-            <ZoomEventLayer
-              focusable
-              height={height}
-              onDoubleClick={onDoubleClick}
-              onKeyDown={onKeyZoom}
-              onPointerEnter={onPointerEnter}
-              onPointerDown={panning ? _zoom.dragStart : DEFAULT_HANDLER}
-              onPointerMove={panning ? _zoom.dragMove : DEFAULT_HANDLER}
-              onPointerUp={panning ? _zoom.dragEnd : DEFAULT_HANDLER}
-              onPointerLeave={onPointerLeave}
-              onWheel={throttledOnWheel}
-              panning={panning}
-              tabIndex={0}
-              width={width}
-            />
-          </ZoomingComponent>
+          <g ref={wheelEventLayer}>
+            <ZoomingComponent
+              initialTransformMatrix={_zoom.initialTransformMatrix}
+              move={move}
+              transformMatrix={_zoom.transformMatrix}
+              transform={_zoom.toString()}
+              {...props}
+            >
+              <ZoomEventLayer
+                focusable
+                height={height}
+                onDoubleClick={onDoubleClick}
+                onKeyDown={onKeyZoom}
+                onPointerEnter={onPointerEnter}
+                onPointerDown={panning ? _zoom.dragStart : DEFAULT_HANDLER}
+                onPointerMove={panning ? _zoom.dragMove : DEFAULT_HANDLER}
+                onPointerUp={panning ? _zoom.dragEnd : DEFAULT_HANDLER}
+                onPointerLeave={onPointerLeave}
+                onWheel={throttledOnWheel}
```

Comment:
The zooming UX is much smoother if you remove `onWheel` here. However, 13 tests rely on it being present, so removing it fails CI.

Reply:
Tunning the zoom parameters in `wheelDelta` also produces a much smoother UX, so I've done that.